src/OpenLoco/src/Objects/VehicleObject.h

```diff
@@ -150,9 +150,8 @@ namespace OpenLoco
         none = 0U,
         hasSprites = 1U << 0,         // If not set then no body will be loaded
         rotationalSymmetry = 1U << 1, // requires half the number of sprites i.e. 32 instead of 64
-        hasUnkSprites = 1U << 2,
-        hasGentleSprites = 1U << 3, // for gentle slopes
-        hasSteepSprites = 1U << 4,  // for steep slopes
+        hasGentleSprites = 1U << 3,   // for gentle slopes
```

Comment:
I would keep the flag Mark as deprecated and say it used to be here and it doesn't do anything. Then we don't accidently reuse it which might cause issues if the dat was used in vanilla

Reply:
I was planning to reuse it as the "cargo draws as child" flag, which the only thing that happens in vanilla is it will draw only the cargo sprite instead of car and cargo together.

Reply:
Why wouldn't you use 7

Reply:
I wasn't clear on discord, removing the code is fine but I agree with Duncan, the flag should be left and marked as deprecated. It can be fully removed in new object format.

Reply:
> Why wouldn't you use 7

its the only flag left and I want as many flags to play with as possible